draft-ietf-quic-transport.md

```diff
@@ -2486,7 +2486,16 @@ A client will always know whether the server has Handshake keys
 whether the client has Handshake keys.  Under these circumstances, a server
 SHOULD send a CONNECTION_CLOSE frame in both Handshake and Initial packets
 to ensure that at least one of them is processable by the client.  These
-packets can be coalesced into a single UDP datagram (see {{packet-coalesce}}).
+packets can be coalesced into a single UDP datagram; see {{packet-coalesce}}.
```

Comment:
Re-reading this paragraph I realize it needs an update for the new timing of discarding Handshake on the server side, because it means a client may want to send a CONNECTION_CLOSE in both Handshake and 1-RTT.

Reply:
#3293

Reply:
thx